gateway/coprocess_bundle.go

```diff
@@ -259,14 +264,29 @@ func (gw *Gateway) fetchBundle(spec *APISpec) (Bundle, error) {
 		return bundle, err
 	}
 
-	bundleData, err := getter.Get()
+	bundleData, err := pullBundle(getter)
 
 	bundle.Name = spec.CustomMiddlewareBundle
 	bundle.Data = bundleData
 	bundle.Spec = spec
 	return bundle, err
 }
 
+func pullBundle(getter BundleGetter) ([]byte, error) {
+	var bundleData []byte
+	var err error
+	downloadBundle := func() error {
+		bundleData, err = getter.Get()
+		return err
+	}
+
+	exponentialBackoff := backoff.NewExponentialBackOff()
+	exponentialBackoff.Multiplier = 0.5
```

Comment:
This value seems wrong, usually you'd want to retry with a larger multiplier, making each next request slower